prow/config/plugins.yaml

```
@@ -187,6 +187,7 @@ milestone_applier:
     release-1.7: "IPAM - v1.7"
   metal3-io/ironic-image:
     main: "ironic-image - v28.0"
```

```suggestion
    main: "ironic-image - v29.0"
```
This now needs to point to v29.0 too.